docs/release-process.md

```diff
@@ -8,73 +24,38 @@
  2. Package two days before a release. So if the release is to be on Thursday,
     package on Tuesday. Think about timezones as well.
 
- 3. Ensure that the tests on Travis CI are green.
+ 3. Ensure that the relevant tests on CI are green.
 
-    See: https://travis-ci.org/php/php-src/builds
+    See: 
+      https://travis-ci.org/php/php-src/builds
```

Comment:
I think we should drop the references to Travis CI right away. It is basically no longer usable, and for PHP 7.4+ we have setups on Azure. We may want to consider to add references to AppVeyor CI (https://ci.appveyor.com/project/php/php-src) instead.